tavern/config.go

```diff
@@ -99,16 +102,20 @@ func (cfg *Config) Connect(options ...ent.Option) (*ent.Client, error) {
 	return ent.NewClient(append(options, ent.Driver(drv))...), nil
 }
 
+func GetBindAddress() string {
```

Comment:
We don't need to create this function, it doesn't do anything except call a different function.